File: jvCPP/1901.cpp
// Series reto 03
// https://jv.umsa.bo/problem.php?id=1901

#include "iostream"
using namespace std;
int main(){

    int cases;
    cin >> cases;

    while(cases--){
        int i = 1;
        int cnt = 0;
        int n, k;
        cin >> n >> k;
        while(cnt < n){
            for (int j = 0; j < k; j++){
                if(cnt < n){
                    cout << i << " ";
                    cnt +=1;
                }   
            }
            i += 2;
        }
        cout << endl;
    }
    return 0;
}

File: jvPY/1901.py
# Series reto 03
# https://jv.umsa.bo/problem.php?id=1901

cases = int(input())

for _ in range(cases):
    i = 1
    cnt = 0
    n, k = map(int, input().split(" "))
    while cnt < n:
        for j in range(k):
            if cnt < n:
                print(i, end=" ")
                cnt +=1
        i += 2
    print()